fe/fe-core/src/main/java/com/starrocks/http/rest/LoadAction.java

```diff
@@ -105,7 +109,20 @@ public void executeWithoutPassword(BaseRequest request, BaseResponse response) t
             throw new DdlException("No backend alive.");
         }
 
-        TNetworkAddress redirectAddr = new TNetworkAddress(backend.getHost(), backend.getHttpPort());
+        String redirectHost = backend.getHost();
+        if (Config.stream_load_force_use_ip) {
+            InetAddressValidator validator = InetAddressValidator.getInstance();
+            if (!validator.isValidInet4Address(redirectHost) && !validator.isValidInet6Address(redirectHost)) {
```

Comment:
It may return an internal IP if the backend has many IPs @QingdongZeng3 
From my opinion, the host is a better solution for security.